src/index.js

```diff
@@ -145,6 +145,18 @@ ReactDOM.render(
             src: require('./components/CommentComposer/docs.md')
           },
           {
+            path: '/components/comment-teaser',
+            title: 'CommentTeaser',
+            imports: {
+              t,
+              CommentTeaser: require('./components/CommentTeaser/CommentTeaser'),
+              CommentTeaserHeader: require('./components/CommentTeaser/CommentTeaserHeader'),
+              CommentTeaserFooter: require('./components/CommentTeaser/CommentTeaserFooter'),
+              CommentHeader: require('./components/Comment/CommentHeader')
+            },
```

Comment:
In CommentTree I created a file `docs.imports.js` so that I don't have to edit this main index file each time I create a new component and want to document it in the catalog. That trick might be useful here, too, if more components are added.

Reply:
Cool, I've moved the imports to docs.imports.js now.